### ISSUE TITLE
Gradle stack

### ISSUE THREAD

**Issue by airazabal** [appsody/stacks#459]
I would like to have a gradle based stack for my java/springboot applications

the alternative is to run the maven stack and run gradle with maven...but personally I don't like that option. If you are a gradle user, you don't mix and match.

**Comment by skoh7645** [comment]
@airazabal yes, we would definitely want to add more stacks so users wouldn't have to "mix and match", we just haven't had time to create them ourselves. That said, we would be happy to guide and help you out if you wanted to create this one?

**Comment by airazabal** [comment]
I have created a working one, although it needs some work to make it usable...see here https://github.ibm.com/irazabal/gradle-stack under incubator. I would appreciate some comments.

**Comment by skoh7645** [comment]
@airazabal It'd be easier for us to review if you push what you have to a PR and we made the comments there  🙂
